src/EPR.Accreditation.Facade.Common/Dtos/Portal/MaterialWasteOutputsDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EPR.Accreditation.Facade.Common.Dtos.Portal
```

Comment:
What's this one all about? We already have a MaterialOutputDto that covers all these properties